en/debug/eclipse_jlink.md

```diff
@@ -0,0 +1,83 @@
+# MCU Eclipse/J-Link Debugging for PX4
+
+This topic explains how to setup and use [MCU Eclipse](https://gnu-mcu-eclipse.github.io/) with a *Segger Jlink adapter* to debug PX4 based-devices.
```

Comment:
this says to debug "PX4" based devices, but  does this mean Pixhawk devices, or specifically PX4 or NuttX?